docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -77,6 +94,16 @@ components:
 
         These credentials must be pre-registered with the service provider.
 
+    AuthClientForwardedUser:
+      type: apiKey
+      in: header
+      name: X-Forwarded-User
+      description: >
+        May only be used in conjunction with AuthClient authentication. Allows
+        requests with AuthClient authentication to perform actions "on behalf of"
+        users within the registered authority. The value of this header should be
+        the userid of the intended forwarded user, e.g. `acct:username@myauthority.net`
```

Comment:
> the userid of the intended forwarded user

I find myself wondering what "forwarded user" means but the sentence above makes it clear so it's probably ok. This could just be me not being familiar with the terminology.